src/apps/core/services/spreadsheets/spreadsheets.py

```diff
@@ -1,57 +1,27 @@
-import asyncio
-from typing import Any
+from http import HTTPStatus
 
-from aiogoogle import Aiogoogle, HTTPError
+import aiohttp
 from django.conf import settings
 
-from .auth import creds
 from .logger import logger
 
 
-async def send(
-    spreadsheetid: str,
-    table_values: list[list[Any]],
-) -> None:
-    """Отправляет переданные данные в Google таблицы.
+class AsyncGoogleFormSubmitter:
+    def __init__(self, form_id: str = settings.GOOGLE_FORM_ID, form_fields: dict = settings.GOOGLE_FORM_FIELDS):
+        """Инициализация URL и полей формы."""
+        self.form_url = settings.GOOGLE_FORM_URL.format(form_id)
+        self.form_fields = form_fields
 
-    Args:
-        spreadsheetid (str): ID Google таблицы
-        table_values (list[list[Any]]): Значения колонок
+    async def submit_form(self, data: dict) -> bool:
+        """Отправка формы."""
+        # Передаются только data ключи совпадающие с ключами в form_fields
+        payload = {f"entry.{self.form_fields[key]}": data[key] for key in self.form_fields.keys() & data.keys()}
 
-    Raises:
-        HTTPError: Не удалось отправить сроку в таблицу
-    """
-    async with Aiogoogle(service_account_creds=creds) as aiogoogle:
-        range = f"1:{len(table_values)}"
-
-        update_body = {"majorDimension": "COLUMNS", "values": table_values}
-        try:
-            service = await aiogoogle.discover("sheets", "v4")
-            await aiogoogle.as_service_account(
-                service.spreadsheets.values.append(
-                    spreadsheetId=spreadsheetid, range=range, valueInputOption="USER_ENTERED", json=update_body
+        # Отправляем POST-запрос на сервер с помощью aiohttp
+        async with aiohttp.ClientSession() as session:
+            async with session.post(self.form_url, data=payload) as response:
+                logger.info(
+                    f"Отправленная форма {self.form_url} со статусом {response.status}/n" f"С данными {payload}"
                 )
```

Comment:
Вообще лучше логгировать так:
```suggestion
                logger.info(
                    f"Отправленная форма %s со статусом %s/nС данными %s",
                    self.form_url, response.status, payload,
                )
```